main.cc

```diff
@@ -288,6 +288,21 @@ static void tcp_syncookies_sanity() {
     }
 }
 
+static void tcp_timestamps_sanity() {
+    try {
+        auto f = file_desc::open("/proc/sys/net/ipv4/tcp_timestamps", O_RDONLY | O_CLOEXEC);
+        char buf[128] = {};
+        f.read(buf, 128);
+        if (sstring(buf) != "1\n") {
+            startlog.warn("sysctl entry net.ipv4.tcp_timestamps is set to 0.\n"
+                          "For better performance and avoid packet loss, set following parameter on sysctl is strongly recommended:\n"
```

Comment:
This does not avoid packet loss. The option makes performance suffer less in a presence of packet loss.